sdk/clientcore/core/src/main/java/io/clientcore/core/credentials/oauth/AccessToken.java

```diff
@@ -11,12 +11,6 @@
  * </p>
  *
  * <p>
- * An Access Token is a security token that is issued by an authentication source, such as
- * Azure Active Directory (AAD), and it represents the authorization to access a specific resource or service.
- * It is typically used to authenticate and authorize requests made to Azure services.
- * </p>
- *
- * <p>
  * Access Tokens are obtained through the authentication process, where the user or application presents valid
  * credentials (such as a client ID, client secret, username/password, or certificate) to the authentication source.
  * The authentication source then verifies the credentials and issues an Access Token, which is a time-limited token
```

Comment:
Because we're moving to a world where we aren't advertising things like username/password can we change this?


```suggestion
 * Access Tokens are obtained through the authentication process, where the user or application presents valid
 * credentials (either a secret or a managed identity) to the authentication source.
 * The authentication source then verifies the credentials and issues an Access Token, which is a time-limited token
```